client/clientmanager/src/main/java/io/mosip/registration/clientmanager/repository/GlobalParamRepository.java

```diff
@@ -3,26 +3,70 @@
 import io.mosip.registration.clientmanager.dao.GlobalParamDao;
 import io.mosip.registration.clientmanager.entity.GlobalParam;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 import javax.inject.Inject;
 
 public class GlobalParamRepository {
 
-    private GlobalParamDao globalParamDao;
+    private static final String TAG = GlobalParamRepository.class.getSimpleName();
+
+    private static Map<String, String> globalParamList;
```

Comment:
initialize this map otherwise, we will hit NPE on call to below getter methods before sync.

Reply:
done